cloudevents/sdk/event/base.py

```diff
@@ -16,6 +16,23 @@
 import json
 import typing
 
+_ce_required_fields = {
+    'ce-id',
+    'ce-source',
+    'ce-type',
+    'ce-specversion'
+}
+
+
+# TODO add datacontenttype?
```

Comment:
Use the optional fields as described in the spec.

https://github.com/cloudevents/spec/blob/master/spec.md#optional-attributes